code-scanning/properties/sysdig-scan.properties.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Sysdigh Inline Scan",
+    "creator": "Sysdig",
+    "description": "Performs analysis on locally built container image and posts the results in SARIF report",
+    "iconName": "sysdig",
```

```suggestion
    "iconName": "sysdig-scan",
```
For ease of keeping track, it might be preferable to call this the same as the workflow itself...?